internal/command/apps/open.go

```diff
@@ -48,11 +48,11 @@ func runOpen(ctx context.Context) error {
 
 	app, err := client.FromContext(ctx).API().GetAppCompact(ctx, appName)
 	if err != nil {
-		return fmt.Errorf("failed retrieving app %s: %w", appName, err)
+		return fmt.Errorf("Failed retrieving app %s: %w", appName, err)
```

Comment:
Just a nitpick, but IIRC typical Go style is to leave the messages for `error`s [uncapitalized](https://google.github.io/styleguide/go/decisions.html#error-strings). I think `flyctl` *generally* adheres to this—but not always, and these are top-level errors anyway, so if you think it looks better for the user, then that's fine by me :smile: